protos/sources/vega/data/v1/spec.proto

```diff
@@ -19,6 +19,10 @@ message PropertyKey {
   string name = 1;
   // type is the type of the property.
   Type type = 2;
+  // An optional decimal place to be be applied on the provided value
+  // valid only for PropertyType of type DECIMAL and INTEGER
```

Comment:
is it actually relevant for decimal?

Reply:
I think it is but I don't think it's handled correctly (or at all), I'll comment about it somewhere else.

Reply:
It shouldn't be. The decimal places is relevant for integer values (e.g. settlement price for ETH is expresset in gwei -> decimal places should be specified so we can scale the oracle value to wei correctly). If the data is provided in decimals (ie settlement asset in USD, 1.1 or 1.23) just have to be scaled by multiplying the decimal by 10^asset precision. Combining number of decimal places on a decimal value makes very little sense. The only thing this could possibly do is truncate decimals from the source, which doesn't sit well with me